lib/dispatch.go

```diff
@@ -30,6 +39,22 @@ func (inst *Instance) Dispatch(ctx context.Context, method string, param interfa
 	// If the http rpc layer is engaged, use it to dispatch methods
 	// This happens when another process is running `qri connect`
 	if inst.http != nil {
+		if tok := token.FromCtx(ctx); tok == "" {
+			// If no token exists, create one from configured profile private key &
+			// add it to the request context
+			// TODO(b5): we're falling back to the configured user to make requests,
+			// is this the right default?
+			p, err := profile.NewProfile(inst.cfg.Profile)
```

Comment:
I'm starting to really like this dispatch setup we have now.
As for the default I'd say yes, the owner is responsible for it's co-profiles in most normal use cases. 
The only alternative being to fail hard and fast here to prevent unintended user switcheroos.